backend/Techfesia2019/event_registrations/models.py

```diff
@@ -38,24 +75,56 @@ def save(self, *args, **kwargs):
 
 
 class TeamMember(models.Model):
-    team = models.ForeignKey(to=Team, on_delete=models.CASCADE)
+    team = models.ForeignKey(to=Team, on_delete=models.CASCADE, related_name='teammember_set')
 
-    profile = models.ForeignKey(to=Profile, on_delete=models.PROTECT)
+    profile = models.ForeignKey(to=Profile, on_delete=models.PROTECT, related_name='member_teams')
 
     invitation_accepted = models.BooleanField(default=False,
-                                              help_text="If true, person has accepted invitation and is part of the team"
+                                              help_text="If true person has accepted invitation and is part of the team"
+                                              )
+
```

Comment:
Why 2 fields?
We can just delete the invitation object if the person rejects it.
Any specific reason?

Reply:
I can make the changes.

Reply:
Sure.

Reply:
https://adwait-thattey.github.io/Techfesia2019/api/0.2.0/docs_0.2.0.html#request-teams-view-team-invitations
actually, it was in docs to show rejected invitations

Reply:
I see. In that case, let it be